src/azure-cli/azure/cli/command_modules/role/_help.py

```diff
@@ -161,7 +161,16 @@
 helps['ad app permission add'] = """
 type: command
 short-summary: Add an API permission
-long-summary: Invoking "az ad app permission grant" is needed to activate it
+long-summary: >-
+    Invoking "az ad app permission grant" is needed to activate it.
+
```

Comment:
Is leaving 2 empty lines here a common pattern for long summary ?

Reply:
YAML will fold 2 empty lines as 1 empty line if `>` is used. See [YAML spec - Line Folding](https://yaml.org/spec/1.2/spec.html#id2779048).